jp/manifestly_deploys.md

```diff
@@ -14,6 +14,9 @@ While fine to try this, this was not the intent of the `upload` action; `upload`
 
 The proposal in this note is to provide a new manifestly action for recording where manifests are deployed.
 
+Why store this in `deploy-manifests`? As opposed to appending to a file on the server (that is backed up)?
```

Comment:
Because `deploy-manifests` (github, history-preserving, etc) is I think more stable than our EC2 machines (can be wiped at any moment, aren't backed up that I know about)?  Also makes sense to me to co-locate the manifest usage info next to the manifests, also means that manifestly has one place that it uses to store and retrieve this info.  

I do recommend exporting the information to each server's "rev.txt" file, FWIW.